README.md

```diff
@@ -1,6 +1,394 @@
 # CNS Object Broker
-## For dynmically provisioning object storage compliant with an S3 interface.
 
 ## WARNING!
+The work in this project is a proof of concept and not intended for production.
 
-The work in this project is a proof of concept and not intended for anything more than that.
+## Utilize Kubernetes Service-Catalog to dynamically provision CNS Object Storage.
+
+## Overview
+A core feature of the Kubernetes system is the ability to provision a diverse
+offering of block and file storage on demand.
+This project seeks to demonstrate
+that by using the Kubernetes-Incubator's [Service-Catalog](https://github.com/kubernetes-incubator/service-catalog),
+it is now also possible to bring dynamic provisioning to S3 object storage.
```

Comment:
do you want to say "S3 _compatible_ object storage" ? Since it's not the real s3...

Reply:
sounds good